From fe9c6a17ced14bb338afff26276dea76c79ad851 Mon Sep 17 00:00:00 2001
From: Charles Short <charles.short@windriver.com>
Date: Mon, 29 Nov 2021 20:52:34 +0000
Subject: [PATCH] Remove openstackclient

Remove build-Depends-Indep for python-openstackclient as it is
not being used and it is causing problems with the build-pkgs
tool.

Signed-off-by: Charles Short <charles.short@windriver.com>

[Ported to python-ironicclient v5.10.0-4 @ Epoxy]
Signed-off-by: jbarboza <joaopedro.barbozalioneza@windriver.com>
---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 3867c43..7320b67 100644
--- a/debian/control
+++ b/debian/control
@@ -21,7 +21,6 @@ Build-Depends-Indep:
  python3-fixtures,
  python3-jsonschema,
  python3-keystoneauth1,
- python3-openstackclient,
  python3-openstackdocstheme,
  python3-openstacksdk,
  python3-osc-lib,
-- 
2.34.1

